### ISSUE TITLE
cheaty@centurix broken under Cinnamon 3.8.7/LinuxMint 19

### ISSUE THREAD

**Issue by perette** [linuxmint/cinnamon-spices-applets#1887]
```
 * Applet version 0.0.2/last-edited 1519201184,
 * Cinnamon version: Cinnamon 3.8.7
 * Distribution - Mint 19
 * Intel
 * Snippet of ~/.cinnamon/glass.log: Nothing relevant
 * Snippet of ~/.xsession-errors: Below
 ```

@centurix

**Issue**
cheaty@centurix ("Cheaty") does not launch.

Initially crashed immediately with error about assignment to undeclared variable; modify line 264 to add 'let':
```
function resolveHome(path) {
        let home = GLib.get_home_dir();  /// This guy
        return path.replace('~', home);
}
```

Cheaty now launches and appears in the Cinnamon panel, but with an empty menu.

Xsession errors are now:
```
Cjs-Message: 10:57:37.049: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.050: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.051: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.051: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.052: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.053: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.053: JS LOG: [LookingGlass/info] Exception: ReferenceError: assignment to undeclared variable contents
Cjs-Message: 10:57:37.057: JS LOG: [LookingGlass/info] Loaded applet cheaty@centurix in 34 ms

```

**Steps to reproduce**
Reload applet: dbus-send --session --dest=org.Cinnamon.LookingGlass --type=method_call /org/Cinnamon/LookingGlass org.Cinnamon.LookingGlass.ReloadExtension string:'cheaty@centurix' string:'APPLET'

**Expected behaviour**
Cheaty launches and there is stuff in its menu.

**Other information**
Appears to be a casualty of the JavaScript interpreter's strictness increasing.

**Comment by Centurix** [comment]
Thanks! I'll get to it this weekend.

**Comment by Centurix** [comment]
Resolved in #1905 pending approval and merge
